site/en/docs/devtools/css/reference/index.md

```diff
@@ -44,6 +44,12 @@ There are many ways to select an element:
 
 ## View CSS {: #view }
 
+### View invalid, overridden, inactive, and other CSS {: #css-issues }
```

Comment:
Looking at the CSS class names, some classes are grey out as well. Should we explain it in this docs?

![ScreenFlow](https://user-images.githubusercontent.com/5917927/203169422-0040f9f4-57af-4607-98b8-9deacc99f8c7.gif)